kt/godot-library/src/main/kotlin/godot/gen/godot/ARVRController.kt

```diff
@@ -37,21 +38,21 @@ import kotlin.Suppress
  * The position of the controller node is automatically updated by the [godot.ARVRServer]. This makes this node ideal to add child nodes to visualize the controller.
  */
 @GodotBaseType
-open class ARVRController : Spatial() {
+public open class ARVRController : Spatial() {
```

Comment:
Is it a change from Kotlin Poet new version ?

Reply:
yeah. I did not touch anything about the api gen